assets/stylesheets/overall.scss

```diff
@@ -57,6 +58,9 @@ body {
         margin-right: 15px;
     }
 }
+#footer-legal {
+    font-size: x-small;
+}
```

Comment:
I seems we don't have a common understanding what the issue is. I don't want the text to be smaller.

I tried to make that clear in https://progress.opensuse.org/issues/157576 with the screenshot and the explaining text. The issue is not that the existing content takes too much space but that on a page with a lot of white space still there is a scrollbar when there shouldn't be one at all. Please see ![https://progress.opensuse.org/attachments/download/17461/Screenshot_20240320_084634_openqa_cropped_version_string.png](https://progress.opensuse.org/attachments/download/17461/Screenshot_20240320_084634_openqa_cropped_version_string.png) again

Reply:
I get that part but I didnt like how it arrange the two div. I thought that the size wouldnt be a big problem. Anyway i remove it and left it as is

Reply:
Sure, you can introduce such changes but then please make them explicit with an according explaining commit message. Also I suggest in pull requests describe the changes that you do in updates. E.g. "I added the missing `display:flex` and added another commit to make the div arrangement look nice by reducing the text size which I think is better but tell me what you think"